docs-v2/integrations/google-security-review.mdx

```diff
@@ -0,0 +1,216 @@
+- Keep the restricted scopes count minimal and user-scoped.
+    - If you ask for 10+ restricted scopes (anecdotal), or domain-wide delegation, Google may ask you to become CASA Tier 3 verified.
+    - CASA Tier 3 is exponentially more work and costly, compared to Tier 2
+- Test users are permanent: You cannot reset the counter or remove test users. Make sure you use the 100 test users wisely.
+
+## Step 2: Prepare for review
+
+For App verification, make sure you fulfill all of [Google's verification requirements](https://support.google.com/cloud/answer/13464321?hl=en&ref_topic=13460882&sjid=4120114855682251882-EU).
+
+Both the "Brand verification requirements" and "Sensitive and Restricted Scope Requirements" are relevant (unless you only use non-sensitive scopes).
+
+<Tip>
+    Good preparation greatly accelerates your review process.
+    
+    Google takes 24-48h to answer emails. A good demo video and precise answers will save you days of back-and-forth with your reviewer.
+</Tip>
+
+Most importantly:
+* Switch to a [custom callback URL in Nango](/guides/api-authorization/whitelabel-the-oauth-flow#use-a-custom-callback-url), and verify the domain of your callback URL in your Google Cloud project.
+* Make sure your contact details in the Google Cloud console project are up to date. All communication is sent there!
+* Prepare a [demo video](https://support.google.com/cloud/answer/13464321?hl=en&ref_topic=13460882&sjid=4120114855682251882-EU#:~:text=2.%20App%20functionality%20demonstration%20video) exactly as requested by Google (shows end to end flow, including OAuth flow, etc.)
+* For each scope you request, prepare a [detailed justification](https://support.google.com/cloud/answer/13464321?hl=en&ref_topic=13460882&sjid=4120114855682251882-EU#:~:text=Request%20narrowest%20scopes) which includes:
+    * An explanation why narrower scopes would not work, including specifics on what functionality would not work as intended. Read Google's sample justification.
+    * This and the demo video are the most frequent sources of back-and-forth communication.
+
+### Additional prep for CASA Tier 2 assessment
+
+CASA assessments have different tiers, but the only relevant one for you here is Tier 2.
+
+#### Run a security pre-scan
+We recommend running a first security scan before you start your CASA assessment. This lets you fix obvious issues now, without the additional back-and-forth hassle with your CASA vendor.
+
+CASA recommends [Fluidattacks](https://fluidattacks.com/) for static security scans. There is a free, open-source CLI that spins up a pre-configured docker container to run a static (SAST) scan against your codebase. [Instructions are here](https://appdefensealliance.dev/casa/tier-2/ast-guide/static-scan).
+
+Ideally, aim for a clean output, but clear at least high and medium threats. Otherwise you will have at least 2 scan cycles during your actual assessment.
```

Comment:
```suggestion
Ideally, aim for a clean output, but clear at least of high and medium threats. Otherwise you will have at least 2 scan cycles during your actual assessment.
```